nextcloudappstore/core/models.py

```diff
@@ -106,6 +106,7 @@ class Meta:
         verbose_name = _('App Release')
         verbose_name_plural = _('App Releases')
         unique_together = (('app', 'version'),)
+        ordering = ['-version']
```

Comment:
AFAIK the ordering field is translated to Sql. ORDER BY for ints is fine, however for chars I think it works on comparing char for char. 

Lets go for 13.0.0 and 2.0.0: First it compares 1 and 2. Since 1 is lower than 2, the string comparison will return that the 13.0.0 version is lower than 2.0.0.

Can you try it out :)

Also check https://blog.codinghorror.com/sorting-for-humans-natural-sort-order/